app/processing/delinked-statement/get-d365-for-delinked-statement.js

```diff
@@ -0,0 +1,25 @@
+const db = require('../../data')
+const config = require('../../config').processingConfig
+
+const getD365ForDelinkedStatement = async (transaction) => {
+  return db.d365.findAll({
+    lock: true,
```

Comment:
Why are we looking to lock the on read ?

Reply:
The locking and skip-locking was used because we routinely have messages arrive out of sequence